docs/pages/eas/metadata/faq.md

```diff
@@ -0,0 +1,56 @@
+---
+title: EAS Metadata - FAQ
+sidebar_title: FAQ
+---
+
+import { Callout } from '~/ui/components/Callout';
+
+<Callout type="warning">
+  EAS Metadata is in beta and subject to breaking changes.
+</Callout>
+
```

Comment:
I like the idea of this being in a FAQ, we should format it like our other FAQ using collapsing sections. Especially if we implement [improved indexing](https://developers.google.com/search/docs/advanced/structured-data/faqpage) in the components.

Reply:
cc @Simek